CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Added mixin pattern rule in package.json.
```

Comment:
Move this to the other "Changed" section.

Reply:
Resolved in 9034343